### ISSUE TITLE
Issue with arrays in function-url-scheme-whitelist

### ISSUE THREAD

**Issue by aaronkahlhamer** [stylelint/stylelint#2446]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug

> Which rule, if any, is this issue related to?

function-url-scheme-whitelist

> What CSS is needed to reproduce this issue?

```css
a {
  background-image: url('data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs=');
}

```

> What stylelint configuration is needed to reproduce this issue?

```json
"function-url-scheme-whitelist": ["https", "data"]
```

> Which version of stylelint are you using?

7.9.0 but the issue can be recreated in the demo at https://stylelint.io/demo/

> How are you running stylelint: CLI, PostCSS plugin, Node API?

gulp-stylelint

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

To loop through the array and ignore both https and data.

> What actually happened (e.g. what warnings or errors you are getting)?

When running the following in the demo...

```css
a {
  background-image: url('data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs=');
}
```

```json
"function-url-scheme-whitelist": ["https", "data"]
```

...it reports, Unexpected url scheme "data:"

**Comment by aaronkahlhamer** [comment]
Screenshot of error...

![image](https://cloud.githubusercontent.com/assets/695639/24373270/68974622-12f6-11e7-8a08-7a1616f8a47f.png)

**Comment by hudochenkov** [comment]
Thank you for catching this bug!

**Comment by hudochenkov** [comment]
Done in https://github.com/stylelint/stylelint/pull/2447

**Comment by aaronkahlhamer** [comment]
Thank you!
